tests/data/test_open_images.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from pathlib import Path
+
+import numpy
+import pytest
+import torch
+from torchvision.transforms import ToTensor
+from utils import create_deterministic_image, write_image_to_file
+
+from neuralcompression.data import OpenImagesV6
+
+
+@pytest.fixture
+def open_images_datafolder(tmp_path: Path):
+    rng = numpy.random.default_rng(0xFEEEFEEE)
+
+    root = tmp_path / "open_images"
+
+    def write_folder_and_flist(base_dir: Path, subfolder: str):
+        directory = base_dir / subfolder
+        directory.mkdir(parents=True)
+
+        n = int(rng.integers(1, 16, (1,)))
+
+        images = []
+
+        with open(base_dir / f"list-{subfolder}-files.txt", "w") as f:
```

Comment:
To be adapted if you adapt the code to provide arbitrary split paths

Reply:
Made a note for the future!